CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.16)
 project(jsonschema VERSION 4.3.0 LANGUAGES CXX)
 list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake")
 include(vendor/noa/cmake/noa.cmake)
-
+include_directories(vendor/termcolor/include)
```

Comment:
Note `termcolor` is a CMake project with its own `CMakeLists.txt`, so instead of including its files directly here (which is not a best practice), you should create a `cmake/FindTermColor.cmake` and them here do `find_package(TermColor REQUIRED)` like we do for other dependencies

Reply:
Alright @jviotti got your point.